tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/LogInstrumentation/Controllers/TestContextController.cs

```diff
@@ -44,9 +44,12 @@ public string Get(string message, string contextData)
         private Dictionary<string, object> GetDictionaryFromString(string data)
         {
             var dict = new Dictionary<string, object>();
-            foreach (var item in data.Split(','))
+            if (data != null)
             {
-                dict[item.Split('=')[0]] = item.Split('=')[1];
+                foreach (var item in data.Split(','))
```

Comment:
Not for this PR, but we should probably make a helper function for this, since a bunch of the logging tests need to do this